sdk/build.gradle

```diff
@@ -33,6 +34,9 @@ dependencies {
     implementation devDependencies.constraintLayout
     implementation devDependencies.picasso
     implementation devDependencies.kotlin
+    implementation ('org.kin.ecosystem.appreciation.options.menu:ui:1.0.0') {
+        transitive = false
```

Comment:
why do you need it?

Reply:
because `options-menu` repo is using the `base` module from here and this resolved duplications.